src/Illuminate/Collections/Collection.php

```diff
@@ -425,6 +425,24 @@ public function get($key, $default = null)
         return value($default);
     }
 
+    /**
+     * Get an item from the collection by key or add it to collection if it does not exist.
+     *
+     * @param  mixed  $key
+     * @param  mixed  $value
+     * @return mixed
+     */
+    public function getOrPut($key, $value)
+    {
+        if (array_key_exists($key, $this->items)) {
```

Comment:
Why you don't use 'has` method here?

Reply:
Because `has` allow an array as `$key` but `get` and `put` not.